### PR TITLE
Update parent

### PR DESCRIPTION
**Pull request by jglick**
Supersedes #239. Matches https://github.com/jenkinsci/workflow-cps-plugin/pull/682.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    ignore:
+    # Must match transitive version from org.jenkins-ci.plugins:subversion:
+    - dependency-name: org.tmatesoft.svnkit:svnkit-cli
```
Comment (jtnord) [inline .github/dependabot.yml:11]: then why don't we just depend on the version transitively?  are their conflicts - or is there a `depenencyManagement` that is lost due to [MNG-5761](https://issues.apache.org/jira/browse/MNG-5761)?

**Review by jtnord — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (jglick) [reply .github/dependabot.yml:11]: > then why don't we just depend on the version transitively?

`subversion` depends only on `svnkit` (an old version) and does not pass along the `svnkit-cli` dep.

> are [there] conflicts

Yes, which is why #239 failed.

Comment (jglick) [reply .github/dependabot.yml:11]: We could also pick up https://github.com/jenkinsci/subversion-plugin/pull/276 via https://github.com/jenkinsci/bom/pull/1672 but that would still be a bit too old. Would need to go up to https://github.com/jenkinsci/subversion-plugin/pull/278 via https://github.com/jenkinsci/bom/pull/1789. :eyes:

**Review by jtnord — APPROVED** [review]
either way works, I was just wondering if there was a way we can minimize the human interaction needed going forward.

**Comment by jglick** [comment]
Unfortunately it does not appear that way. See note in https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-4.55.

**Comment by jglick** [comment]
(see #238)

**Pull request merged**